Source/Icebreaker/Helpers/IcebreakerBotDataProvider.cs

```diff
@@ -195,7 +196,7 @@ public async Task<UserInfo> GetUserInfoAsync(string userId)
         }
 
         /// <summary>
-        /// Get the stored information about given users
+        /// Get the stored opt-in information about given users
```

Comment:
NIT: not a bad change but why is it part of this PR?

Reply:
There's no super good reason that it's in this PR, I just wanted to change it because it's very similar to the summary for GetUserInfoAsync right above it even though they pull different information